docs/libs.md

```diff
@@ -1,19 +1,31 @@
 ---
 id: libs
-title: Libraries
+title: Extensions & Libraries
 ---
-## Angular2+
+## Extensions
+
+These extensions are provided by KaTeX.
+
+- [Auto-render](autorender.md) Automatically renders all of the math inside text
+- [Copy-tex](https://github.com/Khan/KaTeX/tree/master/contrib/copy-tex) When selecting and copying KaTeX-rendered elements, copies their LaTeX source to the clipboard
+- [`math/tex` Custom Script Type](https://github.com/Khan/KaTeX/tree/master/contrib/mathtex-script-type) Automatically displays LaTeX math inside `script` tags with `type=math/tex`
```

Comment:
Can you add colons after each of the names (e.g. `[Auto-render](autorender.md):`)?